main.cpp

```diff
@@ -241,6 +494,32 @@ struct CarWash //                                   1) define an empty struct fo
     3) loop sound
  */
 
+struct Sampler       
+{
+    //has sample slot
+    int sampleSlot = 127; //3) member variables with relevant data types.
+    //has start
+    int start = 127; //3) member variables with relevant data types.
+    //has loop
+    int loop = 127; //3) member variables with relevant data types.
+    //has lenght
+    int lenght = 127; //3) member variables with relevant data types.
+    //has fade
+    int fade = 127; //3) member variables with relevant data types.
```

Comment:
What kind of information are these properties supposed to represent?
again with the copy/paste/rename.   Be more explicit when you name your properties and choose your property types so they convey some actual information.   `startSample`  `loopLength`  `isLooped` `fadeLengthInSamples`  or `fadeLengthInMilliseconds` for example.  Help the reader understand what your code is supposed to do.
- [ ] revise.  pick properties that provide information and name them accordingly.